homotopy-graphics/src/style.rs

```diff
@@ -0,0 +1,22 @@
+use homotopy_core::Generator;
+
+pub trait GeneratorStyle {
+    fn label(&self) -> Option<String>;
+    fn shape(&self) -> Option<VertexShape>;
+    // TODO(thud): migrate color-related code to here for consistency
```

Comment:
Yes, please do that!

Reply:
I think it would be possible to merge the `SignatureStylesheet` functionality into this trait which would make the codebase more consistent and clean.

Reply:
Yes, that would be great. Let's do that in a future PR.